apps/web/pages/apps/installed.tsx

```diff
@@ -96,122 +95,96 @@ function ConnectOrDisconnectIntegrationButton(props: {
-              <IntegrationListItem
-                key={item.title}
-                imageSrc={item.imageSrc}
-                title={item.title}
-                description={item.description}
-                actions={
-                  <ConnectOrDisconnectIntegrationButton
-                    credentialIds={item.credentialIds}
-                    type={item.type}
-                    isGlobal={item.isGlobal}
-                    installed={item.installed}
-                  />
-                }
-              />
-            ))}
-          </List>
-
-          <ShellSubHeading
-            className="mt-10"
-            title={
-              <SubHeadingTitleWithConnections title={"Others"} numConnections={data?.other?.numActive || 0} />
-            }
-          />
-          <List>
-            {data.other.items.map((item) => (
-              <IntegrationListItem
-                key={item.title}
-                imageSrc={item.imageSrc}
-                title={item.title}
-                description={item.description}
-                actions={
-                  <ConnectOrDisconnectIntegrationButton
-                    credentialIds={item.credentialIds}
-                    type={item.type}
-                    isGlobal={item.isGlobal}
-                    installed={item.installed}
-                  />
-                }>
-                <AppConfiguration type={item.type} credentialIds={item.credentialIds} />
-              </IntegrationListItem>
-            ))}
-          </List>
-        </>
-      )}></QueryCell>
+      success={({ data }) => {
+        const installedApps = data.items.filter(filterInstalled);
+        return (
+          <>
+            {installedApps.length > 0 && (
+              <div className={className}>
+                <ShellSubHeading
+                  title={
+                    <SubHeadingTitleWithConnections title={t(variant)} numConnections={data.numActive} />
+                  }
+                />
+                <List>
+                  {installedApps.map((item: AppOutput) => (
+                    <IntegrationListItem
+                      key={item.title}
+                      title={item.title}
+                      imageSrc={item.imageSrc}
+                      description={item.description}
+                      actions={
+                        <ConnectOrDisconnectIntegrationButton
+                          credentialIds={item.credentialIds}
+                          type={item.type}
+                          isGlobal={item.isGlobal}
+                          installed
+                        />
+                      }
+                    />
+                  ))}
+                </List>
+              </div>
+            )}
+          </>
+        );
+      }}></QueryCell>
   );
-}
+};
 
 function Web3Container() {
   const { t } = useLocale();
-
+  const result = trpc.useQuery(["viewer.web3Integration"]);
+  const isWeb3Active = result.data?.isWeb3Active as boolean;
```

Comment:
```suggestion
  const isWeb3Active = !!result.data?.isWeb3Active;
```
If you want to convert it to strick boolean you can use double negation

Reply:
What @alannnc said